releasenotes/notes/qasm-lexer-bugfix-1779525b3738902c.yaml

```diff
@@ -0,0 +1,5 @@
+fixes:
+  - |
+    Fixes qasm lexer so the parser errors if version other than 2.0 is passed.
+    `#139 in openqasm <https://github.com/Qiskit/openqasm/issues/319>` for more
```

Comment:
I've transferred this issue into Terra, so the issue link is out-of-date, but in this case the buggy behaviour is sufficiently simple, we'd be ok without a link.